src/ripple/app/ledger/Ledger.h

```diff
@@ -413,30 +413,19 @@ pendSaveValidated(
     bool isSynchronous,
     bool isCurrent);
 
-extern std::shared_ptr<Ledger>
-loadByIndex(std::uint32_t ledgerIndex, Application& app, bool acquire = true);
+struct SQLLedgerInfo;
 
 extern std::tuple<std::shared_ptr<Ledger>, std::uint32_t, uint256>
```

Comment:
These functions should already have external linkage. No need for the `extern`.

Reply:
Done.